hangman.py

```diff
@@ -0,0 +1,427 @@
+        letters_in_secret_word = []
+        for letters in secret_word:
+            if letters in letters_in_secret_word:
+                continue
+            letters_in_secret_word.append(letters)
+        return letters_in_secret_word
+
+    letters_in_secret_word = letters_in_secret_word(secret_word)
+    list_of_en_golos = ["a", "e", "i", "o", "u"]
+    print("Welcome to the game Hangman!")
+    print("I am thinking of a word that is", len(secret_word), "letters long.")
+    print("You have 3 warnings left")
+    guesses_left = 6
+    warnings_left = 3
+    letters_guessed = []
+    while True:
+        print("--------------------------------------")
+        if guesses_left <= 0:
+            print("Sorry, you ran out of guesses. The word was", secret_word)
+            break
+        print("You have", guesses_left, "guesses left")
+        get_available_letters(letters_guessed)
+        print("Available letters:", get_available_letters(letters_guessed))
+        guessed_letter = input("Please guess a letter: ")
+
+        if guessed_letter.isalpha():
+            guessed_letter = guessed_letter.lower()
+        else:
+            if warnings_left == 0:
+                guesses_left -= 1
+                print("Oops! That is not a valid letter. You have no warnings left so you lose one guess:",
+                      get_guessed_word(secret_word, letters_guessed))
+                continue
+            else:
+                warnings_left -= 1
+                print("Oops! That is not a valid letter. You have", warnings_left, "warnings left:",
+                      get_guessed_word(secret_word, letters_guessed))
+                continue
+
+        if guessed_letter in letters_guessed:
+            if warnings_left == 0:
+                guesses_left -= 1
+                print("Oops! You've already guessed that letter. You have no warnings left so you lose one guess:",
+                      get_guessed_word(secret_word, letters_guessed))
+                continue
+            else:
+                warnings_left -= 1
+                print("Oops! You've already guessed that letter. You have", warnings_left, "warnings left:",
+                      get_guessed_word(secret_word, letters_guessed))
+                continue
+        else:
+            letters_guessed.append(guessed_letter)
+
+        if guessed_letter in letters_in_secret_word:
+            print("Good guess:", get_guessed_word(secret_word, letters_guessed))
+        else:
+            if guessed_letter in list_of_en_golos:
+                guesses_left -= 2
+            else:
+                guesses_left -= 1
+            print("Oops! That letter is not in my word:", get_guessed_word(secret_word, letters_guessed))
+
+        if is_word_guessed(secret_word, letters_guessed):
+            total_score = guesses_left * score(secret_word, letters_guessed)
+            print("Congratulations, you won! Your total score for this game is:", total_score)
```

Comment:
Could you have a finish code to be separate from game iteration? I mean - out of this loop

Reply:
done